cms/djangoapps/contentstore/views/tests/test_item.py

```diff
@@ -145,6 +146,11 @@ def test_container_get_query_count(self, branching_factor, unit_queries,):
         with check_mongo_calls(unit_queries):
             self.client.get(reverse_usage_url('xblock_container_handler', self.populated_usage_keys['vertical'][-1]))
 
+    def test_container_handler(self):
+        response = self.client.get(reverse_usage_url('container_handler', "i4x://UAx/Test101/vertical/static/story2.html"))
+        self.assertEqual(response.status_code, 404)
+
+
```

Comment:
remove one line space as standard is to have one line space between two functions